src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -160,14 +160,57 @@ class ActivityFeed extends React.Component<Props, State> {
         versionInfoHandler(data);
     };
 
+    /**
+     * Determine whether or not a sort should occur, based on new comments, tasks, versions.
+     *
+     * @param {Comments} - [comments] - Object containing comments for the file.
+     * @param {Tasks} - [tasks] - Object containing tasks for the file.
+     * @param {FileVersions} - [versions] Object containing versions of the file.
+     * @return {boolean} True if the feed should be sorted with new items.
+     */
+    shouldSortFeedItems(comments?: Comments, tasks?: Tasks, versions?: FileVersions): boolean {
+        if (!comments || !tasks || !versions) {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     *  If the file has changed, clear out the feed state.
+     *
+     * @param {BoxItem} [file] The box file that comments, tasks, and versions belong to.
+     * @return {boolean} - True if the feedItems were emptied.
+     */
+    clearFeedItems(file?: BoxItem): boolean {
+        const { file: oldFile } = this.props;
+        if (file && file.id !== oldFile.id) {
+            this.setState({ feedItems: [] });
+            return true;
+        }
+
+        // Also check to see if the feedItems list is empty.
+        const { feedItems } = this.state;
+        return !feedItems.length;
```

Comment:
Can you improve the comment here to explain why this is necessary?

Reply:
done

Reply:
Argh, made the change in https://github.com/box/box-ui-elements/pull/332. that PR is tightly coupled with this one, so I'd rather leave the change there instead of reverting and redoing tests